src/theme/index.js

```diff
@@ -8,9 +8,10 @@ export { paletteLight, pictureLight, gridLight, paletteDark, pictureDark, gridDa
 export const getTheme = (isDarkTheme) =>
   createTheme({
     colorSchemes: {
-      light: { palette: paletteLight, picture: pictureLight, grid: gridLight },
-      dark: { palette: paletteDark, picture: pictureDark, grid: gridDark },
+      light: { palette: paletteLight },
+      dark: { palette: paletteDark },
     },
-
+    picture: isDarkTheme ? pictureDark : pictureLight,
+    grid: isDarkTheme ? gridDark : gridLight,
```

Comment:
Is this change there on purpose? I think you reverted some changes of the main branch during a rebase 😅